lake/commit/actions/actions.go

```diff
@@ -71,3 +73,16 @@ func (d *Delete) CommitID() ksuid.KSUID {
 func (d *Delete) String() string {
 	return "DEL " + d.ID.String()
 }
+
+type AddX struct {
```

Comment:
I think AddIndex would be a clearer name.

Reply:
My bad.  I agree.